frontend/components/unified/UnifiedAuctionsView.vue

```diff
@@ -71,112 +86,94 @@ export default Vue.extend({
                         participate: '/debt',
                     },
-                    filters: ['debt'],
                 },
                 {
-                    title: 'Vault liquidations portal',
-                    description:
-                        'Web tool that supports liquidation of vaults that are no longer collaterlised enough',
+                    title: 'Vault liquidations',
+                    description: 'Web tool that supports liquidation of vaults that are not collaterlised enough',
                     links: {
-                        source: 'https://github.com/sidestream-tech/unified-auctions-ui',
                         participate: '/vaults',
                     },
-                    filters: ['collateral'],
                 },
+            ],
+            legacyToolList: [
                 {
-                    title: 'Liquidations platform',
-                    description: 'Web tool that support participation in collateral auctions by bidding with own Dai',
+                    title: 'Collateral liquidation analytics',
+                    description: '',
                     links: {
-                        source: 'https://github.com/makerdao/liquidations-portal',
+                        source: 'https://info.sky.money/liquidations',
+                    },
+                },
+                {
+                    title: 'Liquidation analytics on Makerburn',
+                    description: `
+                    Shows various statistics on current state of the system like current Dai supply,
+                    as well as collateral parameters and information on past liquidations`,
+                    links: {
+                        source: 'https://makerburn.com/#/liquidations',
                     },
-                    filters: ['collateral'],
                 },
                 {
-                    title: 'Collateral Auction Twitter Bot',
-                    description: `Tweets about collateral auctions and facilitates access to auction participation`,
+                    title: 'Source code of the exchange callees',
+                    description: '',
                     links: {
-                        profile: 'https://x.com/sidestream_labs',
+                        source: 'https://github.com/makerdao/exchange-callees',
                     },
-                    filters: ['collateral'],
                 },
                 {
-                    title: 'MakerBurn',
-                    description: `
-                    Shows various statistics on current state of the system like current Dai supply,
-                    as well as collateral parameters and information on past liquidations`,
+                    title: 'Source code of the legacy liquidations-portal UI',
+                    description: 'Web tool that support participation in collateral auctions by bidding with own Dai',
```

Comment:
- [ ] I don't see a value add to include this link. The Github Repo even has been archived. So would suggest to remove even if from UI perspective the last quadrant is then empty. But better than creating noise

Reply:
Replaced it with the link to twitter account. + Added some description text to each link.